lib/cipher/symmetric/hierocrypt3.ts

```
@@ -0,0 +1,352 @@
+}
+
+/**
+ * Key schedule: derive round keys using the cipher's own XS-box construction.
+ * Supports 128/192/256-bit keys.
+ */
+function keySchedule(keyBytes: number[]): number[][] {
+    const rounds = keyBytes.length === 16 ? 6 : keyBytes.length === 24 ? 7 : 8
+    const roundKeys: number[][] = []
+
+    // Expand key via XS-box-based feedback
+    let current = [...keyBytes]
```

ESLint

[error] 'current' is never reassigned. Use 'const' instead.

(prefer-const)